tests/testsuite/config.rs

```diff
@@ -1059,7 +1070,13 @@ Caused by:
   could not parse input as TOML
 
 Caused by:
-  dotted key attempted to extend non-table type at line 2 column 15",
+  TOML parse error at line 3, column 1
+  |
+3 | ssl-version.min = 'tlsv1.2'
+  | ^
+Dotted key `ssl-version` attempted to extend non-table type (`string`)
```

Comment:
```suggestion
Dotted key `ssl-version` attempted to extend non-table type (string)
```

Reply:
Oh and this one is in our control.  I think I did this just for consistency but fine with changing it

Reply:
I had missed updating this.  It is now addressed.